CHANGELOG.md

```diff
@@ -73,6 +73,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   with brackets. To disable, set the new `ui.unique-prefixes` option to `none`
 
 * `jj print` was renamed to `jj cat`. `jj print` remains as an alias.
+  
+* In content that goes to the terminal, the ANSI escape byte (0x1b) is replaced
+  by a "🌈" character. That prevents them from interfering with the ANIS escapes
```

Comment:
There's a typo: ANIS

Also, rainbows are neat, but (I hesitate to say) there's also the boring ␛ character. Another boring thing to do would be to write `\033` as inverted text, like less.

Reply:
Feel free to stick with the 🌈 or to change it 

I thought a bit of there's a better place to put emoji (somewhere we already print Unicode, perhaps), and couldn't think of anything. Having jj log change it's output on Apr 1 would be a nightmare for tests.

Reply:
"␛"sounds good, though the text is so small I have to copy and paste it somewhere else to figure out what it says. I guess it doesn't really matter if users can read it because they're more likely to understand what " 🌈" means anyway. I'll change it. I had actually googled "unicode escape character" before and only found 0x1b :)

Reply:
I'm also unsure how many fonts lack support for these symbols. I tried the default (bitmap?) font xterm starts with on my machine and, surprisingly, it does support "␛".